packages/checkout-full/core/src/components/plug-checkout-full/partials/plug-checkout-full-footer/plug-checkout-full-footer.tsx

```diff
@@ -1,16 +1,34 @@
-import { Component, Host, h, Prop } from '@stencil/core'
+import { Locale } from '@plug-checkout/i18n/dist/utils'
+import { Component, Host, h, Prop, Event, EventEmitter } from '@stencil/core'
 
 @Component({
   tag: 'plug-checkout-full-footer',
   styleUrl: 'plug-checkout-full-footer.scss',
 })
 export class PlugCheckoutFullFooter {
   @Prop() description: string
+  @Prop() language: string
+
+  @Event() changeLanguage: EventEmitter<{ value: Locale }>
+
+  private handleChangeLanguage = (language: Locale) => {
+    this.changeLanguage.emit({ value: language })
+  }
 
   render() {
     return (
       <Host class={{ 'plug-checkout-full-footer__container': true }}>
         <footer class={{ 'plug-checkout-full-footer__content': true }}>
+          <checkout-dropdown
```

Comment:
Não entendi pq tem um dropdown de idioma no footer.

Reply:
É isso mesmo, quando for mobile devemos renderizar o dropdown no footer